Vwitter/UserCell.m

```diff
@@ -0,0 +1,102 @@
+//
+//  UserCell.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/11/22.
+//
+
+#import "UserCell.h"
+#import "Follow.h"
+
+@implementation UserCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    // Initialization code
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+
+    // Configure the view for the selected state
+}
+
+- (void)setUser:(PFUser *)user{
+    _user = user;
+
+    [self refreshData];
+
+}
+
+- (void)refreshData {
+
+    NSString *at = @"@";
+    self.username.text = [NSString stringWithFormat:@"%@%@", at, self.user.username];
+    
+    if ([self.user.objectId isEqualToString:[PFUser currentUser].objectId]) {
```

Comment:
Great job with this!!! One nasty quirk of Objective-C is that you can't do `stringA` == `stringB`, because strings are just objects, and when you do `==` with two objects, all it does is check to see if they are the same _pointer_. So you are doing great to compare using the `isEqualToString:` method. If it were `NSNumber` (an object wrapper of a number) you'd use `isEqualToNumber` and so on 👍